src/CacheKey.php

```diff
@@ -174,7 +174,10 @@ protected function getValuesFromWhere(array $where) : string
 
     protected function getValuesFromBindings(array $where, string $values) : string
     {
-        if (($this->query->bindings["where"][$this->currentBinding] ?? false) !== false) {
+        // Fallback to this when the current binding does not exist in the bindings array
+        $bindingFallback = __CLASS__ . ':UNKNOWN_BINDING';
+
+        if (($this->query->bindings["where"][$this->currentBinding] ?? $bindingFallback) !== $bindingFallback) {
```

Comment:
Thanks for submitting this. I see the problem you are trying to solve, thanks for catching that!

I would prefer to use `data_get()` to fetch the binding, if you can update the PR to that, I'll go ahead and merge.

Reply:
This way of retrieving the current binding was used in multiple places in the class so I extracted it to a private function.